doc/source/ecosystem.md

```diff
@@ -0,0 +1,27 @@
+(ecosystem)=
+
+# Ecosystem
+
+xDEM is but a single tool among a large landscape of open tools for geospatial elevation analysis! Below is a list of 
+other **tools that you might find useful to combine with xDEM**, in particular for retrieving elevation data or to perform complementary analysis.
```

Comment:
Add geoutils, as it might not be evident how closely related these two projects are.

Reply:
Good idea, done!